Microsoft.Toolkit/Diagnostics/Generated/Guard.Collection.tt

```diff
@@ -1,4 +1,5 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+﻿<#@ output extension=".g.cs"#>
```

Comment:
👍 awesome, like the .g.cs extension at least for check-in if we can't solve the generation on build as at least it makes it very clear it's a generated file :)

Reply:
Ah, glad you liked the idea! 😄